nimare/meta/utils.py

```diff
@@ -407,55 +427,130 @@ def compute_ale_ma(shape, ijk, kernel):
 
     Parameters
     ----------
-    shape : tuple
-        Shape of brain image + buffer. Typically (91, 109, 91) + (30, 30, 30).
-    ijk : array-like
+    mask : img_like
+        Mask to extract the MA maps shape (typically (91, 109, 91)) and voxel dimension.
+        The mask is applied to the coordinates before creating the kernel_data.
+    ijks : array-like
         Indices of foci. Each row is a coordinate, with the three columns
         corresponding to index in each of three dimensions.
-    kernel : array-like
+    kernels : array-like, or None, optional
```

Comment:
I believe this is a dict now.

Reply:
We're creating the empty dictionary inside `compute_ale_ma` if `use_dict is True`. I think `kernels` will be a 3D array when `use_dict is False` and `sample_sizes is None`